### ISSUE TITLE
Not enough contrast when selecting title or text in search box 

### ISSUE THREAD

**Issue by filmgal** [insanum/obsidian_nord#14]
(copied from Obsidian forum)
Love this theme! Thank you so much for it.

I’m trying to change one thing and I can’t figure it out.
When I select text it highlights with the secondary colour (I believe) which is great. However, that doesn’t seem to apply to the title or the search box.  I’m not sure where or how to specify a different bg colour when these items are selected.

![image](https://user-images.githubusercontent.com/30246308/104998662-99faa080-59f9-11eb-9199-3e1b031c6e79.png)

**Comment by montemartin** [comment]
I've been doing some customizations on the Nord theme for myself. One of the biggest drivers was not being able to tell when text was highlighted in search boxes. 

What I've learned is that something in Obsidian is causing the `--text-selection` variable to shift to a darker shade in areas outside editor panes. It's less noticable in other themes and with other colors, but for some reason is extreme with Nord. 

I haven't been able to figure out what the cause is. I've scoured the CSS in several themes and in app.css and nothing there seems related. Changing the background of CSS `::selection` to a different value works outside editor panes but the color is still darker than normal. It doesn't effect content in editor panes at all. I'm guessing Obsidian is parsing the `--text-selection` value and applying it separately in Prism.
